### PR TITLE
Editor Resize

### LINKED ISSUE #136: Allow for TinyMCE editor pane to be resized
Currently the editor pane appears at the bottom at a fixed height, or in fullscreen mode it covers the entire preview. There should be a way to resize the pane as desired.

See https://wordpress.org/support/topic/suggested-features-2?replies=4#post-8395773 /cc @pingram3541

### PR DESCRIPTION
**Pull request by valendesigns**
Adds vertical resize capability to post editor pane

Fixes #136

### CODE REVIEW

File: js/customize-post-section.js [hunk 1]
```diff
@@ -352,6 +377,92 @@
...
+				if ( windowHeight < editorPane.outerHeight() ) {
+					args.height = windowHeight;
+				}
+
+				preview.css( 'bottom', args.height );
+				editorPane.css( 'height', args.height );
+				editorFrame.css( 'height', args.height - args.components );
+				collapse.css( 'bottom', args.height + 8 );
+
+				if ( 56 > windowHeight - args.height ) {
```
Comment (westonruter) [inline js/customize-post-section.js]: What is 56? Can we assign it to a named variable so we can understand what is being referred to here?

File: js/customize-post-section.js [hunk 2]
```diff
@@ -352,6 +377,92 @@
...
+					collapse.css( 'bottom', mceStatusbar.outerHeight() + 4 );
```
Comment (westonruter) [inline js/customize-post-section.js]: What is 4? Can we assign it to a named variable so we can understand what is being referred to here?

File: js/customize-post-section.js [hunk 3]
```diff
@@ -352,6 +377,92 @@
...
+				}
+
+				if ( windowWidth <= 782 ) {
```
Comment (westonruter) [inline js/customize-post-section.js]: What is 782? Can we assign it to a named variable so we can understand what is being referred to here?

File: js/customize-post-section.js [hunk 4]
```diff
@@ -352,6 +377,92 @@
...
+				}
+
+				if ( windowHeight < editorPane.outerHeight() ) {
+					args.height = windowHeight;
+				}
+
+				preview.css( 'bottom', args.height );
+				editorPane.css( 'height', args.height );
+				editorFrame.css( 'height', args.height - args.components );
+				collapse.css( 'bottom', args.height + 8 );
```
Comment (westonruter) [inline js/customize-post-section.js]: What is 8? Let's assign to a variable so we can know what it is and to avoid `no-magic-numbers`.

Comment (valendesigns) [inline js/customize-post-section.js]: Yeah I figured that was coming when I added all those to the array. I'll sort it.

**Comment by westonruter** [comment]
@pingram3541 could you please advise on appropriate variable names for those magic-number literals? For more information, see http://eslint.org/docs/rules/no-magic-numbers

**Comment by valendesigns** [comment]
@westonruter I added them

**Pull request merged**